src/metabase/automagic_dashboards/core.clj

```diff
@@ -585,12 +698,13 @@
      (->> (reduce (fn [acc selector]
                     (concat acc (-> selector recommendations rules/ensure-seq)))
                   []
-                  [:table :segments :metrics :linking-to :linked-from :tables :fields])
+                  [:table :segments :metrics :linking-to :dashboard-mates :similar-questions
```

Comment:
This seems like a good candidate for having a name. This combination looks meaningful, but I don't know why.